js/repl/ReplOptions.js

```diff
@@ -677,6 +749,11 @@ const styles = {
     maxWidth: "4rem",
     paddingLeft: "0.75rem",
   }),
+  pluginName: css({
+    width: "100%",
+    padding: "0.75rem",
+    marginBottom: "1rem",
```

Comment:
This two sizes might be need to be updated when this PR is included in the docusaurus site (I had to change some sizes in the repl - https://github.com/babel/website/pull/1533)

Reply:
Yeah, I need to change to branch of this PR against the new website.